.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  name: Toitware
+  url: https://www.toitware.com/contact.html
```

Comment:
I wonder if we should change this to community.toitware.com once we have a friendly front for our public Slack offering up and running?

Reply:
That is a great idea!